serviceWorker.js

```diff
@@ -0,0 +1,61 @@
+const CACHE = "root-offline";
```

Comment:
Where does this file come from? Did you write this yourself? If not, what is the license of the file and where does it come from?

Reply:
No it's just a name I came up with of the cache file.